app/Models/User.php

```diff
@@ -36,4 +36,29 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function role()
+    {
+        return $this->belongsTo('App\Models\Role');
+    }
+
+    public function suggests()
+    {
+        return $this->hasMany('App\Models\User');
```

Comment:
Mối quan hệ này là sao nhỉ ?

Reply:
cái này e nhầm ạ, phải là belongsTo mới đúng

Reply:
cái này e viết nhầm ạ, e đã sửa lại return $this->hasMany('Suggest::class');